app/src/main/java/org/gdg/frisbee/android/utils/PrefUtils.java

```diff
@@ -182,11 +183,18 @@ public static void setFatalPlayServiceMessageShown(Context context) {
         prefs(context).edit().putBoolean(PREFS_FATAL_GOOGLE_PLAY_SERVICE, false).apply();
     }
 
-    public static boolean isSummitNotificationSent(Context context) {
-        return prefs(context).getBoolean(PREFS_SUMMIT_NOTIFICATION_SENT, false);
+    public static boolean isTaggedEventSeriesNotificationSet(Context context, TaggedEventSeries taggedEventSeries) {
+        long lastTimeSent = prefs(context).getLong(getKeyForEventSeries(taggedEventSeries), 0);
+        return lastTimeSent >= taggedEventSeries.getStartDateInMillis().getMillis();
```

Comment:
Next year when we increase the `startDateInMillis`, this will automatically return false. I like it. 👍